client/android/app/build.gradle

```diff
@@ -51,6 +53,19 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+
+    flavorDimensions app
+
+    productFlavors {
+        hack {
+            dimension app
+            versionNameSuffix "-hack"
+        }
+        prod {
+            dimension app
+            versionNameSuffix "-prod"
```

Comment:
Add in staging flavor. This will leave the original staging (who-myhealth-staging) as the default.

@paulinar - do you know if the following is the right way to set a default build flavor? We want developers to default to the staging server... then they can get other servers with a specific flavor.

https://proandroiddev.com/android-set-a-default-build-type-flavor-a1cdcd1c15fa
```
productFlavors {
  ...
  staging {
    dimension server
    getIsDefault().set(true)
  }
}
```

Reply:
Added these lines